src/bundle/Form/Type/SearchType.php

```diff
@@ -42,27 +43,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
             ->add('query', CoreSearchType::class, ['required' => false])
-            ->add('page', HiddenType::class)
-            ->add('limit', HiddenType::class)
+            ->add('page', HiddenType::class, ['empty_data' => 1])
+            ->add('limit', HiddenType::class, [
+                'empty_data' => $this->configResolver->getParameter('pagination.search_limit'),
+            ])
             ->add('content_types', ContentTypeChoiceType::class, [
                 'multiple' => true,
                 'expanded' => true,
             ])
             ->add('last_modified', DateIntervalType::class)
             ->add('created', DateIntervalType::class)
-            ->add('creator', UserType::class)
-            ->add('last_modified_select', ChoiceType::class, [
-                'choices' => $this->getTimePeriodChoices(),
-                'required' => false,
-                'placeholder' => /** @Desc("Any time") */ 'search.any_time',
-                'mapped' => false,
-            ])
-            ->add('created_select', ChoiceType::class, [
-                'choices' => $this->getTimePeriodChoices(),
-                'required' => false,
-                'placeholder' => /** @Desc("Any time") */ 'search.any_time',
-                'mapped' => false,
-            ])
+            ->add('search_in_users', SearchUsersType::class, ['property_path' => 'searchUsersData'])
```

Comment:
Could you explain what are the differences in behavior between "site" and AdminUI so in ezsystems/ezplatform-admin-ui#1410 you're removing this field? What exactly does this field allow and why it needs to be removed there? I'm not up to date with the requirements, sorry :)

Reply:
In adminUI there is filtering content by one single user, there is a search in the background and you have those dropdown suggestions.
Since we do not put JS in default front end templates, that solution is kinda now go (we could move to filtering by email/id/login for example, but then it is not useful as user name is displayed everywhere).

So instead of one user filtering this solution narrows filtering to all users that match pattern that was previously used to select single one.